reinforced_lib/agents/base_agent.py

```diff
@@ -0,0 +1,33 @@
+from typing import NamedTuple, Callable
+
+import gym
+
+
+class BaseAgent(NamedTuple):
+    """
+    Container for functions of the agent.
+
+    Fields
+    ------
+    init : Callable
+        Creates and initializes state for ThompsonSampling agent.
```

Comment:
Czy nie powinno to być generyczne? Tutaj ograniczasz zakres jedynie do TS agent

Reply:
Zwykła pomyłka, nie zauważyłem przy przenoszeniu klasy do nowego pliku